src/components/Loader/Loader.tsx

```diff
@@ -1,8 +1,10 @@
-import React from 'react';
+// import React from 'react';
 import './Loader.scss';
 
-export const Loader: React.FC = () => (
-  <div className="Loader" data-cy="loader">
-    <div className="Loader__content" />
-  </div>
-);
+export const Loader = () => {
+  return (
+    <div className="Loader" data-cy="loader">
+      <div className="Loader__content" />
+    </div>
+  );
```

Comment:
Use the short syntax for returning JSX elements in functional components. Replace the return statement and enclosing braces with a direct expression.